torch/csrc/jit/jit_log.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <string>
+
+namespace torch {
+namespace jit {
+
+enum class JitLoggingLevels {
+  OFF,
+  GRAPH_DUMP,
```

Comment:
Nit: I'm not sure these levels would actually be convenient to use(i.e. would I always want to see graph dumps if I'm interested in `info`?), but it works for now :)

Reply:
I think, we can figure out which levels we need and what those levels should log as we go. For now, I was thinking to use `INFO` to report individual transformations (we removed a node, constant folded an expression). It usually makes sense to also have graphs after those transformations handy. You could also filter out `JIT_GRAPH_DUMP` if you only need `INFO` messages without graph dumps. It's not ideal, but it's very simple, though.

Reply:
I am not a fan of options. They are not discoverable, people will mix them in poor ways. I feel like we should either always report the thread ID or never report it. Honestly never is better since the compiler almost always is running in a single thread now.